src/app/friend/friend-list/friend-list.component.scss

```diff
@@ -0,0 +1,10 @@
+@import 'src/app/styles/layout/variable.scss';
+
+//list style
+.expense-list {

```

Comment:
Duplicate style of expanse list if style is common then it should ne common scss file which is accessible to all components.

Reply:
Changes done.